charts/jaeger/templates/agent-svc.yaml

```diff
@@ -28,10 +28,10 @@ spec:
     port: {{ .Values.agent.service.samplingPort }}
     protocol: TCP
     targetPort: http
-  - name: admin
+  - name: http-admin
```

Comment:
isn't this a breaking change ?
If someone is using the named-ports for their serviceMonitors for example it would be an issue.

Reply:
Can we not make the names configurable: `{{ .Values.collector.service.otlp.grpc.name }}` 🤔 
https://github.com/jaegertracing/helm-charts/blob/main/charts/jaeger/values.yaml#L375

Reply:
I think you are right, this will be a breaking change, considering their use for monitoring

Reply:
I will refactor this PR and make those port names configurable, so that it is the best option